src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_x.h

```diff
@@ -368,6 +374,7 @@ namespace sirf {
 			gadgetron::shared_ptr<GadgetronImageData> sptr_ic
 			) : sptr_acqs_(sptr_ac) //, sptr_imgs_(sptr_ic)
 		{
+            sptr_enc_ = std::make_shared<sirf::CartesianFourierEncoding>(sirf::CartesianFourierEncoding());
```

Comment:
```suggestion
            sptr_enc_ = std::make_shared<sirf::CartesianFourierEncoding>();
```
before you were creating an object, and then calling the copy-constructor. Now you just create.

Reply:
done